commands/login.go

```diff
@@ -11,9 +11,9 @@ import (
 	"strings"
 	"time"
 
-	"github.com/openfaas/faas-cli/proxy"
-
 	"github.com/openfaas/faas-cli/config"
+	"github.com/openfaas/faas-cli/faaserrors"
```

Comment:
I think it makes sense to just name the package `errors` and then import at 

```
faaserrors "github.com/openfaas/faas-cli/errors"
```

or

```
clierrors "github.com/openfaas/faas-cli/errors"
```

especially since the only error is not a generic OpenFaaS error statement but a specific cli error

Reply:
@LucasRoesler  clierrors seems like a more sensitive name. Will change